tests/hardhat/integration/index.ts

```diff
@@ -0,0 +1,561 @@
+
+      await prime.accrueInterest(vusdt.address);
+      await prime.accrueInterest(veth.address);
+
+      interestForUser1ForUsdt = await prime.callStatic.getInterestAccrued(vusdt.address, user1.getAddress());
+      interestForUser1ForEth = await prime.callStatic.getInterestAccrued(veth.address, user1.getAddress());
+
+      expect(interestForUser1ForEth).to.gt(0);
+      expect(interestForUser1ForUsdt).to.gt(0);
+
+      // providing some liquidity to PLP
+      await usdt.transfer(primeLiquidityProvider.address, convertToUnit("1", 6));
+      await eth.transfer(primeLiquidityProvider.address, convertToUnit("1", 6));
+      await prime.accrueInterest(veth.address);
+      await prime.accrueInterest(vusdt.address);
+
+      const interestForUser1ForEthIncludingPlp = await prime.callStatic.getInterestAccrued(
+        vusdt.address,
+        user1.getAddress(),
+      );
+      const interestForUser1ForUsdtIncludingPlp = await prime.callStatic.getInterestAccrued(
+        veth.address,
+        user1.getAddress(),
+      );
+      expect(interestForUser1ForEthIncludingPlp).to.gt(interestForUser1ForEth);
+      expect(interestForUser1ForUsdtIncludingPlp).to.gt(interestForUser1ForUsdt);
+
+      const previousBalanceOfUser1ForUsdt = await usdt.balanceOf(user1.getAddress());
+      const previousBalanceOfUser1ForEth = await eth.balanceOf(user1.getAddress());
+
+      await prime.connect(user1)["claimInterest(address)"](vusdt.address);
+      await prime.connect(user1)["claimInterest(address)"](veth.address);
+
+      expect(await usdt.balanceOf(user1.getAddress())).to.be.closeTo(
+        previousBalanceOfUser1ForUsdt.add(interestForUser1ForUsdtIncludingPlp),
+        convertToUnit("1", 3),
+      );
+      expect(await eth.balanceOf(user1.getAddress())).to.closeTo(
+        previousBalanceOfUser1ForEth.add(interestForUser1ForEthIncludingPlp),
+        convertToUnit("1", 3),
+      );
+
+      // Making two other users get the prime token
+      await xvs.connect(user2).approve(xvsVault.address, bigNumber18.mul(10000));
+      await xvsVault.connect(user2).deposit(xvs.address, 0, bigNumber18.mul(10000));
+
+      await xvs.connect(user3).approve(xvsVault.address, bigNumber18.mul(10000));
+      await xvsVault.connect(user3).deposit(xvs.address, 0, bigNumber18.mul(10000));
+
+      await mine(90 * 24 * 60 * 60);
+
+      await prime.connect(user2).claim();
+      await prime.connect(user3).claim();
+
+      // Minting and borrowing by user3
+      await usdt.transfer(user3.getAddress(), convertToUnit("10000", 18));
+      await usdt.connect(user3).approve(vusdt.address, bigNumber18.mul(9000));
+      await vusdt.connect(user3).mint(bigNumber18.mul(9000));
+
+      await comptroller.connect(user3).enterMarkets([vusdt.address, veth.address]);
+
+      await veth.connect(user3).borrow(bigNumber18.mul(1));
+
+      await prime.accrueInterest(vusdt.address);
+      await prime.accrueInterest(veth.address);
```

Comment:
`accrueInterest` is invoked (indirectly) from the Comptroller's hooks on each user interaction with the market. So, I think we should not need to invoke them manually after minting vUSDT and borrowing vETH

Reply:
yes, removed